### DISCUSSION TITLE
Is it possible to have VSCode Sync on, but not sync one specific extension on one specific machine?

### DISCUSSION THREAD

**Discussion by geolectric** [microsoft/vscode-discussions#583] [Extension Development QnA] [answered]
I'm not sure if this is the correct place to ask this, but I couldn't find another VSCode discussion.

I have VSCode Sync turned on for all Settings, and I sync between my personal and work machines, and I have an extension that I use on my personal machine (Stripe) that I'd rather not have on my work machine, but I want all the other settings and extensions to sync.

Is it possible to have the full sync settings turned on but not install one of the extensions on one of the machines?

Thanks

**Comment by alefragnani** [comment, marked as answer]
Hi @dropdx,

Yes, you can choose not only which extensions to sync, but also which settings. Simply follow the steps detailed here https://code.visualstudio.com/docs/editor/settings-sync#_configuring-synced-data

Hope this helps
